tests/integration/kubernetes/tests_common.sh

```diff
@@ -314,5 +314,5 @@ wait_for_blocked_request() {
 
 	command="kubectl describe pod ${pod} | grep \"${endpoint} is blocked by policy\""
 	info "Waiting ${wait_time} seconds for: ${command}"
-	waitForProcess "${wait_time}" "$sleep_time" "${command}" || return 1
+	waitForProcess "${wait_time}" "$sleep_time" "${command}" >/dev/null 2>/dev/null
```

Comment:
I don't fully get this change here, @danmihai1

Reply:
> I don't fully get this change here, @danmihai1

The improved diagnosability when using Regorus this way comes at the expense of messier kubectl output. For examples of messy "kubectl describe <pod>", search for "CreateContainerRequest is blocked by policy" in https://productionresultssa10.blob.core.windows.net/actions-results/834f1908-e474-4be2-a0cf-8c3b1382901d/workflow-job-run-2b16350a-daeb-5112-dd9c-126ea1f33816/logs/job/job-logs.txt?rsct=text%2Fplain&se=2024-04-23T18%3A41%3A08Z&sig=rX2I1EZqLm%2BevdBFYESD3yyFq%2BuswZbpu9SeOJ%2BLCxQ%3D&sp=r&spr=https&sr=b&st=2024-04-23T18%3A31%3A03Z&sv=2021-12-02

Before this change, we were displaying the same "CreateContainerRequest is blocked by policy" messy output at least two times:
- First time from waitForProcess
- A second time from teardown() of each test

So, I decided to reduce some of that noise by keeping just of teardown() output and hiding here the redundant waitForProcess output.

Reply:
Makes sense, thanks for the explanation, the approval is still valid. :-)